components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java

```diff
@@ -1940,6 +1940,16 @@ public static NativeArray jsFunction_getAPI(Context cx, Scriptable thisObj, Obje
                                     policyObj.put("tierDisplayName", policyObj, policy.getName() != null ? policy.getName() : "");
                                     policyObj.put("tierDescription", policyObj,
                                             policy.getDescription() != null ? policy.getDescription() : "");
+                                    if (policy.getTierAttributes() != null) {
+                                        Map<String, Object> attributes;
```

Comment:
Shall we declare and assign the "attributes" variable in a single line without splitting into 2 lines?